examples/api/test/material/material_state/material_state_border_side.0_test.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/material_state/material_state_border_side.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  void expectBorderColor(Color color) {
+    final Finder finder = find.byWidgetPredicate((Widget widget) {
+      if (widget is! Material) {
+        return false;
+      }
+      final ShapeBorder? shape = widget.shape;
+      if (shape is! OutlinedBorder) {
+        return false;
+      }
+      return shape.side.color == color;
+    });
+    expect(finder, findsOne);
```

Comment:
Please turn the finder instead of void and put the `expect` in the test itself where this would be called. 
This helps readability, one don't have to function to find out the `expect`.

Reply:
Sure, I made the change in [refactor: Return a finder](https://github.com/flutter/flutter/pull/151089/commits/d2095bd05b97f44907fa7b8e79dc00077b19f4f2)